### DISCUSSION TITLE
Welcome to sqlmesh Discussions!

### DISCUSSION THREAD

**Discussion by tobymao** [TobikoData/sqlmesh#587] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anishjoni** [comment]
I am wondering if I can use R transformations with sqlmesh.

My pipeline starts with sqlserver but I do most of my data transformations in r and the feature I want to use sqlmesh for is column level lineage. 

If I can export my transformed data tables as csv, can I make a sqlmesh project with _sql models_ of raw data and connect it with a _seed model_ that has the transformed data tables? 

Will I be able to see the lineage from sql source tables flowing through the transformed tables created by r(potentially loaded as a seed model)?

**Reply by georgesittas** [reply]
Hey @anishjoni, can you make a separate "Discussions" thread about this? I imagine this one won't get any visibility by users of the community.

**Reply by anishjoni** [reply]
sounds good, I'll make a separate post under "Discussions". Thanks!
